python/src/aiconfig/editor/client/src/shared/prompt_schemas/HuggingFaceImage2TextTransformerPromptSchema.ts

```diff
@@ -28,6 +32,16 @@ export const HuggingFaceImage2TextTransformerPromptSchema: PromptSchema = {
         type: "string",
         description: `Hugging Face model to use`,
       },
+      max_new_tokens: {
+        type: "integer",
+        description: `The amount of maximum tokens to generate. 
+        By default it will use \`generate\` default.`,
+      },
+      timeout: {
+        type: "number",
+        description: `The maximum time in seconds to wait for fetching images 
+        from the web. If None, no timeout is set and the call may block forever.`,
+      },
```

Comment:
Oh wow that's a short list of possible params

Reply:
yea it's actually pretty wild. It's why I left a comment about it in in summary for https://github.com/lastmile-ai/aiconfig/pull/855